Automattic/woocommerce-payments
# Displaying Clearpay instead of Afterpay for UK based stores

Fixes #6425

#### Changes proposed in this Pull Request

Based on the account country set in Stripe, we should display Clearpay instead of Afterpay everywhere.





#### Testing instructions

Using this branch and a Stripe account with a UK address access all pages that show payment methods and ensure Clearpay name and logo is used instead of Afterpay. Some pages you should check:

- Checkout
- Wcpay settings
- Admin transactions page
- Order details

Then update to a Stripe account from the US and ensure we show Afterpay instead of Clearpay.

I recommend using the Jurassic Ninja link bellow twice and creating different stripe test accounts.

-------------------


**Post merge**
